src/fread.c

```diff
@@ -1167,8 +1184,7 @@ int freadMain(freadMainArgs _args) {
       if (sep==' ') while (*ch==' ') ch++;  // multiple sep=' ' at the jlineStart does not mean sep(!)
       // detect blank lines ...
       skip_white(&ch);
-      while (*ch=='\r') ch++;
-      if (*ch=='\n' || *ch=='\0') {
+      if (eol(&ch) || *ch=='\0') {
         if (!skipEmptyLines && !fill) break;
         ch += (*ch=='\n');
```

Comment:
what about '\r' here?

Reply:
good spot. should be ch+=(*ch!='\0') then. needs a test, will do.